glide.yaml

```diff
@@ -43,6 +43,10 @@ import:
     version: v1.2.2
     repo:    https://github.com/aws/aws-sdk-go
 
+### Rackspace
+  - package: github.com/rackspace/gophercloud
+    ref:     42196eaf5b93739d335921404bb7c5f2205fceb3
```

Comment:
Hi @akiraw,

In the `glide.yaml` file please use a release tag or branch name whenever possible instead of a commit hash. The `glide.lock` file will have the commit ID, but `glide.yaml` is meant to make things friendlier for the user/developer. Thanks!